x-pack/legacy/plugins/siem/public/components/news_feed/index.tsx

```diff
@@ -39,7 +42,8 @@ export const StatefulNewsFeed = React.memo<{
         if (!canceled) {
           setNews(getNewsItemsFromApiResponse(apiResponse));
         }
-      } catch {
+      } catch (error) {
+        errorToToaster({ title: i18n.NEWSFEED_FETCH_ERROR, error, dispatchToaster });
```

Comment:
Since the SIEM app will function normally if the newsfeed is temporarily unavailable, and per the screenshot below, we provide an error state, consider removing the error toaster for this case.

<img width="1680" alt="newsfeed-error-state" src="https://user-images.githubusercontent.com/4459398/76256285-459d6680-6215-11ea-913c-0dcedfb98bb5.png">

Reply:
👍 will revert

Reply:
https://github.com/elastic/kibana/pull/59438/commits/177490e4eaeb5ff3b1cc64d4643e16d4c260f025